docs/en/observability/apm/feature-roles.asciidoc

```diff
@@ -164,6 +236,8 @@ If you don't use the +apm_system+ user:
 |====
 +
 . Assign the *monitoring role* to users who need to write monitoring data to {es}.
+. Assign <<apm-feature-roles,additional APM feature roles>> to users as needed
+(including the _Central configuration management role_, which is <<apm-central-config-role-note,required in most cases>>).
```

Comment:
As mentioned above, most of the time it needs apm_writer as well. I wonder if we could capture that without cluttering this note.